pandas/tests/reshape/test_pivot.py

```diff
@@ -1638,3 +1639,21 @@ def test_crosstab_tuple_name(self, names):
 
         result = pd.crosstab(s1, s2)
         tm.assert_frame_equal(result, expected)
+
+    @pytest.mark.parametrize("names, input_data, expected_data_out", [
+        (['a', 'b'], [[1, 2, 3], [1, 1, 1]], [1, 1, 1]),
+        ([('a', 'b'), 'c'], [[1, 2, 2], [1, 1, 1]], [1, 2]),
+        ([('a', 'b'), ('c', 'd')], [[1, 2, 3], [1, 2, 3]],
```

Comment:
can u make input_data into 2, call them row_data and col_data)

Reply:
I was hoping for this to be consistent with "names", which is consistent with the test above this test. Is there a reason to split these?